pages/terms.js

```diff
@@ -1,21 +1,29 @@
 import Layout from '../components/layout';
-import { Hero } from '../components/jambonz-ui';
-import { getData } from '../lib/data';
+import { Hero, TextLayout } from '../components/jambonz-ui';
+import { getData, getParsedMarkdown } from '../lib/data';
 
-export default function Terms({ data }) {
+// You'll need to load the nodejs path module
```

Comment:
Overall this looks good. My only nit pick now would be that we don't need these comments (I just had those in my example for clarity sake). Same goes for `privacy.js`.